examples/basic/binomial/runEfficiency.cxx

```diff
@@ -30,8 +30,14 @@ int main()
     // close log file
     BCLog::CloseLog();
 
+    // models can be copied and assigned
```

Comment:
I don't think this test should reside in an example. OR some more explanation of what is being demonstrated to the user is necessary.

Reply:
I agree, I dropped this commit and added `BCFitter_TEST` instead.